static/app/utils/loadPrismLanguage.ts

```diff
@@ -0,0 +1,66 @@
+import prismComponents from 'prismjs/components';
+
+/**
+ * A mapping object containing all Prism languages/aliases that can be loaded using
+ * `loadPrismLanguage`. Maps language aliases (`js`) to the full language name
+ * (`javascript`).
+ */
+export const prismLanguageMap: Record<string, string> = Object.fromEntries(
+  Object.entries(prismComponents.languages)
+    .map(([lang, value]) => {
+      if (!value.alias) {
+        return [[lang, lang]]; // map the full language name to itself
+      }
+
+      return [
+        [lang, lang], // map the full language name to itself
+        ...(Array.isArray(value.alias) // map aliases to full language name
+          ? value.alias.map(alias => [alias, lang])
+          : [[value.alias, lang]]),
+      ];
+    })
+    .flat(1)
+);
```

Comment:
This new mapping object serves 2 purposes: mapping aliases to full language names (so now we can pass aliases like `js` into `loadPrismLanguage` and it'll work), and checking to see if a language/alias is supported by Prism.